packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2690,6 +2726,24 @@ export default mixins(
 
 
 		async mounted () {
+			window.addEventListener('message', async (message) => {
+				try {
+					const json = JSON.parse(message.data);
+					if (json && json.command === 'openWorkflow') {
+						try {
+							await this.importWorkflowExact(json);
+						} catch (e) {
+							this.$showMessage({
+								title: 'Could not import workflow',
+								message: (e as Error).message,
+								type: 'error',
+							});
```

Comment:
can you add post message event here when import fails? so we can easily add behaviour in our webcomponent when an error occurs